client/src/components/pages/common/Home/index.js

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
+import DotsStepper from '../../../application/DotsStepper';
+import registrationSteps from '../../../application/DotsStepper/registrationSteps.json';
+import draw1 from '../../../../assets/images/draw1.svg';
 
-const Home = () => <h1>Home</h1>;
+import './index.css';
 
+const Home = () => (
+  <>
+    <div className="root">
+      <DotsStepper steps={registrationSteps.registrationSteps} />
+
+      <img className="img" src={draw1} alt="background" />
```

Comment:
did you try to use our image component ?? if no, please try it . if it doesn't do what you want then try to make this image as background to be responsive or find another solution for that